### DISCUSSION TITLE
Interview?

### DISCUSSION THREAD

**Discussion by mneumegen** [web-infra-dev/rspress#200] [General]
Rspress is shaping up to be a tool many technical writers have been waiting for. The fast build times and native MDX support are going to be game changers for documentation sites.

I'd love to do a video interview with someone from the team (@sanyuan0704?) about the philosophy behind Rspress, go over some of the features, and what the future looks like. Let me know - mike[at]cloudcannon.com.

**Comment by sanyuan0704** [comment]
Thank you for your invitation. I'm glad to share about the design and features behind Rspress.

**Reply by mneumegen** [reply]
Amazing! I'm really looking forward to talking. Let's schedule a time to talk over email, could you send me a message at mike@cloudcannon.com.

Thanks :-)
